examples/jsm/objects/ReflectorForSSRPass.js

```diff
@@ -230,7 +230,9 @@ var Reflector = function ( geometry, options ) {
 		renderer.state.buffers.depth.setMask( true ); // make sure the depth buffer is writable so it can be properly cleared, see #18897
 
 		if ( renderer.autoClear === false ) renderer.clear();
+		this.visible = false;
 		renderer.render( scene, virtualCamera );
+		this.visible = true;
```

Comment:
I think it should not control its own display or not within the Reflector, and deliberately commented out the original codes.
I still hope that the display of Reflector will be controlled by the outer code.